builder/amazon/common/access_config.go

```diff
@@ -210,9 +210,7 @@ func (c *AccessConfig) NewEC2Connection() (ec2iface.EC2API, error) {
 		return nil, err
 	}
 
-	ec2conn := ec2.New(sess, &aws.Config{
-		HTTPClient: commonhelper.HttpClientWithEnvironmentProxy(),
```

Comment:
I think we might still need that client from the commonhelper.

Reply:
No I think that was my original mistake -- I believe the config we're defining up in line 74 where I changed the code here is being attached to session via the session.Options, so it's already in there when we're calling ec2.New. I think what was happening here is I was overwriting the configuration that we did on the session, and that broke everything _but_ the proxy. I set up a little toy proxy using tinyProxy with this PR and it worked, so I consider that validation that we don't need that commonhelper tool I wrote.

Reply:
ah, nice ! thanks for the explanation :)